backend/src/apiserver/server/artifact_server.go

```diff
@@ -212,6 +215,7 @@ func (s *ArtifactServer) generateResponseArtifact(
 	bucketConfig *objectstore.Config,
 	namespace string,
 	includeShareUrl bool,
+	includeRenderUrl bool,
```

Comment:
Optional, but rather than a bool, you could replace `includeShareUrl` and `includeRenderUrl` with a single argument of type `GetArtifactRequest_ArtifactView` since it doesn't seem like this will ever need to generate more than one. This would allow for additional options in the future.